greenriver/hmis-frontend
# Bulk Services / Bed Nights v2

## V2 Bulk Services & Bed Nights



Depends on hmis-warehouse PR: https://github.com/greenriver/hmis-warehouse/pull/4087

## How to Test
* BED NIGHTS: go to a night-by-night shelter project page
* BULK SERVICES: mark one or more CUstomServiceTypes as "supports_bulk_assignment" in the database, and go to the project page for a project that supports that service
* COC SELECTION: add multiple ProjectCoC records to the project
* UNIT ASSIGNMENT: add units to the project. when you enroll someone, they should be assigned to the next available unit
see also [qa instructions](https://www.pivotaltracker.com/story/show/187011913/comments/240472290)

## Type of change

## Checklist before requesting review
